zerver/tests/test_message_move_topic.py

```diff
@@ -334,7 +334,7 @@ def assert_is_topic_muted(
         # state + 1/user with a UserTopic row for the events data)
         # beyond what is typical were there not UserTopic records to
         # update. Ideally, we'd eliminate the per-user component.
-        with self.assert_database_query_count(27):
+        with self.assert_database_query_count(29):
```

Comment:
I think this reflects a bug; if the channel didn't change, we don't need to fetch anything extra.

Reply:
You are right! Wrapped the new logic in `if message_edit_request.is_stream_edited:`.